internal/serverinstall/nomad.go

```diff
@@ -802,7 +787,12 @@ EVAL:
 		}
 
 		if allocID != "" {
-			return allocID, nil
+			if retries == maxRetries {
+				return allocID, nil
+			} else {
+				s.Update("Ensuring allocation %q has properly started up...", allocs[0].ID)
+				time.Sleep(1 * time.Second)
+			}
```

Comment:
Because we move the WaitIndex forward before trying again, won't the call to Allocations wait more than 1 second if the allocation is running fine, because there won't be any updates to report, so the call won't return?

Reply:
From what I could tell, the call to Allocations uses a default `WaitTime` if none specified, but I can set a more explicit one of 500 milliseconds to be sure. From my testing in the positive case, the call always returned after the ~3 seconds this loop retries on.

Reply:
The change looks good!